### PR TITLE
Add soft fail for buildah warning

### PR DESCRIPTION
**Pull request by b10n1k**
Softfail for https://bugzilla.suse.com/show_bug.cgi?id=1189893
which throws the "Failed to decode the keys [\"storage.options.ostree_repo\"] from \"/etc/containers/storage.conf\"

Workaround commenting out the ostree_repo in /etc/containers/storage.conf

- Related ticket: https://progress.opensuse.org/issues/103296
- Verification run: http://aquarius.suse.cz/tests/7417

### CODE REVIEW

File: lib/containers/common.pm [hunk 1]
```diff
@@ -116,6 +116,10 @@ sub install_buildah_when_needed {
             zypper_call('in buildah', timeout => 300);
         }
     }
+    if ((script_run 'buildah info') != 1) {
```
Comment (pdostal) [inline lib/containers/common.pm:119]: Didn't you thought about `script_output()` validation instead? In my opinion `buildah info` returning `1` can happen for various reasons.

Comment (b10n1k) [reply lib/containers/common.pm]: *this change is ready for review* http://aquarius.suse.cz/tests/7419

Comment (pdostal) [reply lib/containers/common.pm]: Nice! :+1:

**Review by pdostal — APPROVED** [review]
:rocket

**Review by grisu48 — APPROVED** [review]
LGTM :rocket:

**Pull request merged**